src/server/session.ts

```diff
@@ -1231,7 +1232,8 @@ namespace ts.server {
                 this.getDefaultProject(args),
                 { fileName: args.file, pos: position },
                 !!args.findInStrings,
-                !!args.findInComments
+                !!args.findInComments,
+                this.getHostPreferences().usePrefixAndSuffixTextForRename || false

```

Comment:
>this.getHostPreferences().usePrefixAndSuffixTextForRename || false [](start = 16, length = 66)

Since there may be multiple preferences affecting rename behavior (e.g. #28677), would it make more sense to pass in the entire preferences object (e.g. as in `getEditsForFileRename`)?

Reply:
Switched to passing entire preferences object.
However I only passed it as deep as this call; any deeper would require an API change, so I didn't want to do that unless absolutely necessary.